bin/stencil-pull.js

```diff
@@ -22,13 +24,14 @@ const cliOptions = program.opts();
 const options = {
     dotStencilFilePath: DOT_STENCIL_FILE_PATH,
     apiHost: cliOptions.host || API_HOST,
-    saveConfigName: cliOptions.save,
+    saveConfigName: cliOptions.filename,
+    channelId: cliOptions.channel_id || 1,
+    saved: cliOptions.saved || false,
 };
 
-stencilPull(options, (err, result) => {
+stencilPull(options, err => {
     if (err) {
         printCliResultError(err);
-        return;
+        throw err;
```

Comment:
Throwing the error will print the same message as printCliResultError already did (just without the call stack)

Reply:
Good idea - I'll make an update.

Reply:
@MaxGenash @bookernath I think we should update `printCliResultError` to also include the stack if it is available and after logging instead of doing a return we can do exit.

Reply:
@MaxGenash @junedkazi ♻️

Reply:
I opted to not do a  full stack trace, because for these "expected" errors it's not super helpful. It just shows a trace back to the async module.